packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -357,15 +357,15 @@ def export_frame(self) -> Frame:
             }
         )
 
-    async def finish(self, answer: str) -> str:
+    async def submit_answer(self, answer: str) -> str:
         """Finish the task by submitting an answer to the question.
 
         Args:
             answer: The answer to the question.
         """
         self.state.done = True
         if not answer:
-            return "Finish failed. No answer provided."
+            return "submit_answer failed. No answer provided."
```

Comment:
```suggestion
            return "submit_answer failed since it was provided an empty answer."
```

"no" answer is incorrect, it's really "empty" answer